src/chrono/timestepper/ChTimestepperHHT.h

```diff
@@ -122,6 +125,22 @@ class ChApi ChTimestepperHHT : public ChTimestepperIIorder, public ChImplicitIte
     virtual void Advance(const double dt  ///< timestep to advance
                          ) override;
 
+    /// Get an indicator to tell whether the iteration in current step tends to convergence or divergence.
+    /// This could be helpful if you want to fall back to iterate again via using more rigorous stepper settings, 
+    /// such as smaller fixed time stepper, turning off ModifiedNerton,etc, 
+    /// WHEN you are not satisfied by current iteration result.
+    bool GetConvergenceFlag() { return convergence_trend_flag; };
```

Comment:
Please change to:
bool GetConvergenceFlag() const { return convergence_trend_flag; }

Reply:
See commit 03d74a2 please.